### PR TITLE
Stage/Phase Rocket Chip Bump

### PR DESCRIPTION
**Pull request by davidbiancolin**
**Related issue**: ucb-bar/chipyard#503

**Type of change**: other enhancement

**Impact**:  unknown

**Release Notes**
Bump Rocket Chip.

### CODE REVIEW

**Review by jerryz123 — APPROVED** [review]
Merge at your leisure

File: src/main/scala/common/tile.scala [hunk 1]
```diff
@@ -173,7 +173,7 @@ class BoomTile(
       mtvecWritable       = boomParams.core.mtvecWritable
     )
   )
-  val rocketLogicalTree: RocketLogicalTreeNode = new RocketLogicalTreeNode(cpuDevice, fakeRocketParams, dtim_adapter, p(XLen))
+  //val boomLogicalTree = new BoomLogicalTreeNode(this)
```
Comment (abejgonzalez) [inline src/main/scala/common/tile.scala]: Can we delete the "fake" Rocket params?

**Review by abejgonzalez — APPROVED** [review]
Just 1 question on deleting the extra rocket params. Otherwise LGTM.

**Comment by abejgonzalez** [comment]
In a separate PR, Ill go ahead and update this BOOM CI to work with the newer Chipyard.

File: src/main/scala/common/tile.scala (hunk 1 above)
Comment (davidbiancolin) [reply src/main/scala/common/tile.scala]: For sure! I just forgot to do it.

**Pull request merged**